internal/provisioner/plain/controllers/bundleinstance_controller.go

```diff
@@ -45,9 +45,11 @@ import (
 	"sigs.k8s.io/yaml"
 
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	crdValidatorAnnotation "github.com/operator-framework/rukpak/cmd/crdvalidator/annotation"
```

Comment:
nit: this import could be lowercase and more condensed, at the expense of clarity, to be more conformant to go import conventions. Something like `crdanno`?